Sources/AsyncQueue/AsyncQueue.swift

```diff
@@ -32,6 +32,9 @@ public final class AsyncQueue: Sendable {
         let taskStream = AsyncStream<@Sendable () async -> Void> { continuation in
             capturedTaskStreamContinuation = continuation
         }
+        guard let capturedTaskStreamContinuation = capturedTaskStreamContinuation else {
```

Comment:
```suggestion
        guard let capturedTaskStreamContinuation else {
```
Are we good with Swift 5.7 or do you need to support older versions of the language?

Reply:
I keep forgetting this landed!! Yeah good call.

In most repos I write I support back to Xcode 11 in order to get simulators to test on. But, given that Swift concurrency doesn't go back that far, meh. Let's stick to Xcode 14+